examples/movement.py

```diff
@@ -5,7 +5,7 @@
 from cri_lib.robot_state import OperationMode
 
 # 🔹 Configure logging
-logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)  # Retrieves the logger from __init__.py
```

Comment:
This line should remain as it was. It is the right place to _configure_ the logging.
```suggestion
logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
```

Reply:
i have changed the lines for better orientation by setting the basic configuration level first and calling the logger later.